src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/IndexManagementPlugin.kt

```diff
@@ -302,4 +304,8 @@ internal class IndexManagementPlugin : JobSchedulerExtension, NetworkPlugin, Act
     override fun getTransportInterceptors(namedWriteableRegistry: NamedWriteableRegistry, threadContext: ThreadContext): List<TransportInterceptor> {
         return listOf(rollupInterceptor)
     }
+
+    override fun getActionFilters(): MutableList<ActionFilter> {
```

Comment:
Did this need to be mutable list? Or can we change to List

Reply:
good point, looks like I can just do List - will update